test/build-certificate.js

```diff
@@ -1,6 +1,95 @@
 'use strict'
 
-const selfCert = require('self-cert')
+const os = require('os')
+const forge = require('node-forge')
```

Comment:
I missed this and am now seeing it in #5757. Why did we embed this code? @simone-sanfratello @mcollina